src/code42cli/cmds/profile.py

```diff
@@ -152,24 +228,100 @@ def update(
     """Update an existing profile."""
     c42profile = cliprofile.get_profile(name)
 
-    if (
-        not server
-        and not username
-        and not password
-        and disable_ssl_errors is None
-        and use_v2_file_events is None
-    ):
-        raise click.UsageError(
-            "Must provide at least one of `--username`, `--server`, `--password`, `--use-v2-file-events` or "
-            "`--disable-ssl-errors` when updating a profile."
-        )
-    cliprofile.update_profile(
-        c42profile.name, server, username, disable_ssl_errors, use_v2_file_events
-    )
-    if not password and not c42profile.has_stored_password:
-        password = _prompt_for_password(c42profile.name)
-    if password:
-        _set_pw(name, password, debug, totp=totp)
+    if c42profile.api_client_auth == "True":
+        if (
```

Comment:
the `any()` method can make this a bit more readable:

```
if not any([server, api_client_id, secret, disable_ssl_errors, use_v2_file_events]):
    raise click.UsageError(
       ...
```

Reply:
Oh, I guess that won't work if someone passes `--disable-ssl-errors=false`

Reply:
I could at least group them a little I bet